synfig-studio/src/gui/widgets/widget_timeslider.h

```diff
@@ -56,9 +56,18 @@ class Widget_Timeslider: public Gtk::DrawingArea
 
 	Cairo::RefPtr<Cairo::SurfacePattern> play_bounds_pattern;
 
+	Glib::RefPtr<Gdk::Cursor> bounds_cursor = Gdk::Cursor::create(get_display(), "ew-resize");
+	Glib::RefPtr<Gdk::Cursor> default_cursor = Gdk::Cursor::create(get_display(), "default");
+
+	Glib::RefPtr<Gdk::Pixbuf> lower_boundary_pixbuf;
+	Glib::RefPtr<Gdk::Pixbuf> upper_boundary_pixbuf;
+
 	// last mouse position for dragging
 	double lastx;
 
+	bool move_boundary_button_lower=false;
+	bool move_boundary_button_upper=false;
```

Comment:
keep naming consistency: "move_lower_bound_button", "lower_bound_pixbuf" (or change icon name to follow "lower boundary")
I don't know what is more correct in English: bound or boundary though.

Reply:
ok makes sense.